.travis.yml

```diff
@@ -28,7 +28,7 @@ addons:
 
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update           ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install qt4      ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then sh mac-build.sh       ; fi
```

Comment:
the main problem is the brew removed qt4 package.
So i'm trying to install Qt4 by another way ...
actually i thinks migrating to Qt5 is a better way but it's need lots of work i guess